Makefile

```diff
@@ -8,9 +8,9 @@ IMAGE_BUILD_CMD ?= docker build
 IMAGE_BUILD_EXTRA_OPTS ?=
 IMAGE_PUSH_CMD ?= docker push
 CONTAINER_RUN_CMD ?= docker run
-BUILDER_IMAGE ?= golang:1.25-trixie
-BASE_IMAGE_FULL ?= debian:bookworm-slim
-BASE_IMAGE_MINIMAL ?= scratch
+BUILDER_IMAGE ?=
```

Comment:
This we'd need to leave as is because `hack/prepare-release.sh` breaks otherwise